README.md

```diff
@@ -5,7 +5,7 @@ It shall NOT be edited by hand.
 
 # Kresus for YunoHost
 
-[![Integration level](https://dash.yunohost.org/integration/kresus.svg)](https://dash.yunohost.org/appci/app/kresus) ![](https://ci-apps.yunohost.org/ci/badges/kresus.status.svg) ![](https://ci-apps.yunohost.org/ci/badges/kresus.maintain.svg)  
+[![Integration level](https://dash.yunohost.org/integration/kresus.svg)](https://dash.yunohost.org/appci/app/kresus) ![Working status](https://ci-apps.yunohost.org/ci/badges/kresus.status.svg) ![Maintenance status](https://ci-apps.yunohost.org/ci/badges/kresus.maintain.svg)  
```

Comment:
For ****** I hate this Auto-update that creates conflcits and even adds typos.
Can we disable it?

Reply:
It makes reviews harder, and the whitespace things are just a nightmare to understand.